src/index.json

```diff
@@ -66838,6 +66838,59 @@
                     "version": "0.1.11"
                 },
                 "sha256Digest": "cc99134288545178d08b18abd5b7c9e3d099d2add8b52ab9308f6c5fd97ae60c"
+            },
+            {
+                "downloadUrl": "https://hybridaksstorage.z13.web.core.windows.net/SelfServiceVM/CLI/stack_hci_vm-1.0.0.1-py3-none-any.whl",
+                "filename": "stack_hci_vm-1.0.0.1-py3-none-any.whl",
+                "metadata": {
+                    "azext.minCliCoreVersion": "2.15.0",
+                    "classifiers": [
+                        "Development Status :: 4 - Beta",
+                        "Intended Audience :: Developers",
+                        "Intended Audience :: System Administrators",
+                        "Programming Language :: Python",
+                        "Programming Language :: Python :: 3",
+                        "Programming Language :: Python :: 3.6",
+                        "Programming Language :: Python :: 3.7",
+                        "Programming Language :: Python :: 3.8",
+                        "License :: OSI Approved :: MIT License"
+                    ],
+                    "extensions": {
+                        "python.details": {
+                        "contacts": [
+                            {
+                            "email": "azpycli@microsoft.com",
+                            "name": "Microsoft Corporation",
+                            "role": "author"
+                            }
+                        ],
+                        "document_names": {
+                            "description": "DESCRIPTION.rst"
+                        },
+                        "project_urls": {
+                            "Home": "https://github.com/Azure/azure-cli-extensions/tree/master/src/stack-hci-vm"
+                        }
+                        }
+                    },
+                    "extras": [],
+                    "generator": "bdist_wheel (0.30.0)",
+                    "license": "MIT",
+                    "metadata_version": "2.0",
+                    "name": "stack-hci-vm",
+                    "run_requires": [
+                        {
+                        "requires": [
+                            "azure-identity",
+                            "azure-identity",
+                            "azure-mgmt-resourcegraph",
+                            "azure-mgmt-resourcegraph"
+                        ]
+                        }
+                    ],
+                    "summary": "Microsoft Azure Command-Line Tools Stack-HCi-VM Extension",
+                    "version": "1.0.0.1"
```

Comment:
The version should contain three numbers: `major.minor.patch`.

Reply:
Hi @jsntcy, This has been fixed! Could we please get a review on this PR?

Reply:
I think the version should be `1.0.0` as it's the first version starting from `1.x.x`.

Reply:
The version has to be 1.0.1 since we had an internal release with a 1.0.0 whl file. We cannot override that file.